versions.tf

```diff
@@ -17,7 +17,23 @@
 terraform {
   required_version = ">= 0.13"
   required_providers {
-    google      = ">= 3.54.0"
-    google-beta = ">= 3.54.0"
+
+    google = {
+      source  = "hashicorp/google"
+      version = "~> 3.53"
+    }
+    google-beta = {
+      source  = "hashicorp/google-beta"
+      version = "~> 3.53"
```

Comment:
@bharathkkb is there a way to make it start at 3.54?

Reply:
Is there a reason you think 3.54 is needed? We prefer to keep the minimum version lower where possible.

Reply:
Hi @xingao267 
Feel free to edit this PR as needed. This was only meant to be a starting point for module authors to transition.

Reply:
We need a provider fix for healthcare dataset that is released in 3.54 https://github.com/hashicorp/terraform-provider-google/releases/tag/v3.54.0

Reply:
I'll send a commit to update that version in this PR

Reply:
thank you both for the help!